Assets/HoloToolkit/Utilities/Scripts/HashCodes.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
HashCodes isn't necessary for input to work, but we found them useful while developing our app. I'm fine keeping them or removing them. Anyone feel strongly either way?